src/canmatrix/canmatrix.py

```diff
@@ -1136,24 +1311,26 @@ def renameEcu(self, old, newName):
             frame.updateReceiver()
 
     def addEcu(self, ecu):
+        """Add new ECU to the Matrix. Do nothing if ecu with the same name already exists.
```

Comment:
I mixed up Ecu and BU (=BoardUnit) --
One name should be enough.
Which one we should choose?

Reply:
Personally I've never heard 'Board Unit' but always 'ECU'. But it doesn't matter. The abbreviation 'ECU' seems clearer than 'BU' for me. You decide :-)

Reply:
OK, ECU should be the name for it in future.